addons/website/models/website.py

```diff
@@ -1052,8 +1041,7 @@ def _filter_domain(website, domain_name, ignore_port=False):
         elif len(websites) == 1:
             return websites.id
         else:  # > 1 website with the same domain
-            country_specific_websites = websites.filtered(lambda website: country_id in website.country_group_ids.mapped('country_ids').ids)
-            return country_specific_websites[0].id if country_specific_websites else websites[0].id
+            return websites[0].id
```

Comment:
Same remark

Reply:
Fixed, kept the [0] as we can still have multiple website returned in the case of no domain search -> multiple website without domain is allowed